stories/stories.scss

```diff
@@ -1,5 +1,5 @@
 html {
-  font-family: 'SymphonyLato', 'Lato', 'Segoe UI', 'Helvetica Neue', 'Verdana',
+  font-family: 'system-ui', 'Lato', 'Segoe UI', 'Helvetica Neue', 'Verdana',
```

Comment:
Same comment as in styles. The SDS are showing components with "system-ui" so we should do the same.